### ISSUE TITLE
Homebrew formula does not install zkg

### ISSUE THREAD

**Issue by rsmmr** [zeek/zeek#1529]
From a quick look I believe that's because it goes through `cmake` (instead of `configure`) and cmake doesn't set `INSTALL_ZKG` by default.

**Comment by bbannier** [comment]
The "simple fix" seems to be to fix the CMake default of `INSTALL_ZKG` to `on`. While this would install `zkg` as part of the Homebrew install, the formula still has an issue in that it would now depend on the `GitPython` and `semantic-version` Python packages for which no Homebrew formulas exist. I am unsure what the proper fix is.

**Comment by 0xxon** [comment]
I actually feel like that this is not really a problem on our side, or but that is something that whoever maintains the homebrew package should fix.

So - if they call cmake directly, instead of going through our configure script - the homebrew package maintainers have to  update this when new options are introduced.

This feels similar to basically every linux distribution that packages Zeek - a lot of the recent changes that we have done will require changes on their packaging (as they do for our own packaging on OBS).

So - I am not really sure there is anything for us to do here.

**Comment by bbannier** [comment]
@0xxon, yeah, I think I tend to agree. While having the defaults of `./configure` and plain `cmake` align helps to make it easier for users to follow along Zeek defaults and would help catch such "regressions", it doesn't look `INSTALL_ZKG` would be special and the only differing option.

As for a workaround for this issue, installing `zkg` with e.g., `pipx` seems to work fine, at least for me and at least until now.

**Comment by rsmmr** [comment]
Hmm, ok, I guess I didn't  think about that it's not us that's maintaining these. Then we should at least report it upstream I think.

**Comment by 0xxon** [comment]
That's reasonable. I looked into opening an issue on https://github.com/Homebrew/homebrew-core/issues - and stopped since I am not actually using homebrew, they want a bunch of output of brew commands, and they say that they will close your issue if you don't provide them.

**Comment by ckreibich** [comment]
I seem to recall coming across one other setting recently in which somebody isn't using `configure` and going straight to `cmake`, but I can't locate it now. We could either invest the effort to make sure that this works (if it's just about Zeek's `configure`/`cmake` interaction, it strikes me as relatively little work), or go the other way and abort if it's clear that `configure` wasn't run.

**Comment by rsmmr** [comment]
I'll go ahead and close this, agree that it's really up to them to set this up as desired, and given that there's no easy way to provide that feedback, there's not much to do here for us.

Regarding configure/cmake: I'm reluctant to do much there. First, it's hard to ensure that the defaults for configure and make are fully aligned, easy to miss stuff, either now or in the future; but just aborting without configure is probably also not a good solution because some distros may want to insist on using cmake directly.
